cmd/orch-tester/orch_tester.go

```diff
@@ -135,6 +265,13 @@ func main() {
 		bcastHost = *broadcaster
 	}
 
+	var embeddedBcastMetrics *broadcasterMetrics
+	if *streamTester == "" && *broadcaster == "" && *metrics == "" {
+		glog.Infof("Using embedded broadcaster metrics")
+		embeddedBcastMetrics = &broadcasterMetrics{}
```

Comment:
Should probably use the constructor instead of a raw struct value. (or if we never use the constructor, maybe don't even need to have it)

Reply:
Yeah, we don't need it. Removed.

Reply:
Ahh, no you're right, we need the constructor. Added it back and used it.